### DISCUSSION TITLE
some time i get error and compile after 2,3 attempts .

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#689] [General]
Some time after 2,3 attempts my Arduino code compile ,  I don't know what is the reason , may be it has some link with your library . 

![image](https://user-images.githubusercontent.com/65456998/225987274-8a7746b5-95bb-42a5-9c82-3b02ecfea868.png)

**Comment by pschatzmann** [comment]
I guess you use an old version of libhelix!
